admin/src/main/resources/io/buoyant/admin/js/src/router_client.js

```diff
@@ -169,8 +170,14 @@ define([
 
       function toggleClientDisplay(expand) {
         if (expand) {
+          $contentContainer.css({"border": colorBorder});
+          $headerLine.css("border-bottom", null);
+
           metricsCollector.registerListener(metricsHandler, getDesiredMetrics);
         } else {
+          $contentContainer.css({'border': null});
+          $headerLine.css({'border-bottom': colorBorder});
```

Comment:
Hmmmm what about passing the colorBorder? do you prefer that we create a class for each shade and then toggle that?